docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md

```diff
@@ -90,3 +92,17 @@ If Replicated is using HTTP, you need to remove the standbys and delete the HA c
 1. [Configure HA on the updated instances](../../administer-yugabyte-platform/high-availability/#configure-active-and-standby-instances).
 
 [Failovers](../../administer-yugabyte-platform/high-availability/#failover) are possible again after the completion of this step.
+
+### Migrate from Replicated to YBA Installer on a different VM
+
+1. [Install YBA Installer](../install-software/installer/) on a new VM and [configure YBA Installer](../install-software/installer/#configuration-options) appropriately. Note that [Replicated configuration](../install-replicated/#set-up-https-optional) will not be migrated in this method.
+2. Disable any YBA [high availability](../../administer-yugabyte-platform/high-availability/) (HA) configured on the Replicated YBA.
+3. Perform a full backup on the Replicated installation. This backup includes prometheus and all existing releases. If the backup is too large, consider cleaning up unused releases through the YBA UI and/or reducing the prometheus retention interval in Replicated settings.
```

Comment:
Are your referring to https://docs.yugabyte.com/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp/

Reply:
Probably need links to relevant sections for deleting releases and Prometheus retention.